addons/stock/report/stock_forecasted.py

```diff
@@ -65,22 +65,22 @@ def _get_report_header(self, product_template_ids, product_ids, wh_location_ids)
         if product_template_ids:
             products = self.env['product.template'].browse(product_template_ids)
             res.update({
-                'product_templates' : products.read(fields=['id', 'display_name']),
-                'product_templates_ids' : products.ids,
-                'product_variants' : [{
-                        'id' : pv.id,
-                        'combination_name' : pv.product_template_attribute_value_ids._get_combination_name(),
+                'product_templates': products.read(fields=['id', 'display_name']),
```

Comment:
Although I fully agree that it's putting annoying things in your IDE, you should avoid changes like this just for the sake of aesthetics. It messes with history, and especially since your PR is already huge, may add to the confusion :confused: